common-utils/pom.xml

```diff
@@ -30,6 +30,10 @@
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
```

Comment:
Why common-utlis has a dep to groovy-sql ?

Reply:
Because of ConnectionUtils which is using the Sql object.